packages/platform/platform-fastify/test/test-session.integration.spec.ts

```
@@ -0,0 +1,118 @@
+  name: string;
+
+  @Required()
+  @Email()
+  @Allow(null)
+  email: string;
+
+  @Required()
+  @MinLength(6)
+  @Allow(null)
+  password: string;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)